### DISCUSSION TITLE
parameter HDDS_CONTAINER_CHUNK_WRITE_SYNC_KEY

### DISCUSSION THREAD

**Discussion by julienlau** [apache/ozone#7488] [FAQ]
Hello,

I see that the parameter HDDS_CONTAINER_CHUNK_WRITE_SYNC_KEY is false by default.
Is this paramter related to fault tolerance ?
Should we set it to true for a production setup ?

Regards

**Comment by ChenSammi** [comment]
HDDS_CONTAINER_CHUNK_WRITE_SYNC_KEY controls whether each data chunk write sync to disk file. Set it to true will have performance impact.  As Ozone has high level mechanism to guarantee that data is persist to disk before acking the client, so it's fine to have it as false.
